diff --git a/apps/make/Makefile-Linux-x86-64-gcc b/apps/make/Makefile-Linux-x86-64-gcc
index 6f4886a..3e8d306 100644
--- a/apps/make/Makefile-Linux-x86-64-gcc
+++ b/apps/make/Makefile-Linux-x86-64-gcc
@@ -1,5 +1,6 @@
-KDU_GLIBS =
+KDU_GLIBS = -DKDU_INCLUDE_TIFF
 # KDU_GLIBS += -static -static-libgcc # Uncomment this to get a static build
+DEFINES = -DKDU_INCLUDE_TIFF
 
 CC = g++
 
@@ -52,7 +53,7 @@ endif
 
 CFLAGS = $(INCLUDES) $(C_OPT)
 
-LIBS = -lm -lpthread # You may comment out "-lpthreads" if coresys was
+LIBS = -lm -lpthread -ltiff # You may comment out "-lpthreads" if coresys was
                      # built using -DKDU_NO_THREADS
 NETLIBS =
 
