--- /home/tina/v7_9-01574L/apps/make/Makefile-Linux-x86-64-gcc	2017-01-09 06:30:29.000000000 +0100
+++ /home/tina/build_kakadu/v7_9-01574L/apps/make/Makefile-Linux-x86-64-gcc	2017-07-28 15:10:54.000000000 +0200
@@ -1,8 +1,8 @@
-KDU_GLIBS =
-# KDU_GLIBS += -static -static-libgcc # Uncomment this to get a static build
+KDU_GLIBS = -DKDU_INCLUDE_TIFF
+#KDU_GLIBS += -static -static-libgcc # Uncomment this to get a static build
 
 CC = g++
-
+DEFINES = -DKDU_INCLUDE_TIFF
 # The following variables allow you to turn on/off SIMD optimizations
 INCLUDE_SSSE3 = 1 # Comment out to avoid SSSE3; value is not important
 INCLUDE_SSE4 = 1  # Comment out to avoid SSE4.1; value is not important
@@ -52,7 +52,7 @@ endif
 
 CFLAGS = $(INCLUDES) $(C_OPT)
 
-LIBS = -lm -lpthread # You may comment out "-lpthreads" if coresys was
+LIBS = -lm -lpthread  -ltiff # You may comment out "-lpthreads" if coresys was
                      # built using -DKDU_NO_THREADS
 NETLIBS =
 
@@ -63,6 +63,7 @@ STATIC_LIB_SRC = libkdu.a
 SHARED_LIB_SRC = libkdu_v79R.so
 
 LIB_SRC = $(SHARED_LIB_SRC) # Change to $(STATIC_LIB_SRC) for static linking
+#LIB_SRC = $(STATIC_LIB_SRC) # Change to  for static linking
 LIB_IMPORTS = 
 
 include Targets.mk
